### ISSUE TITLE
[Legitimate Site Blocked] discord-accounts.ru

### ISSUE THREAD

**Issue by mmoadvert** [MrLuit/EtherScamDB#1508]
Domain: https://discord-accounts.ru/

Details: Our site poses no danger. We have been working since 2020. We sell clean new automatically registered Discord messenger accounts. According to search engines "google" and "yandex" we have leading positions in CIS countries. Please exclude the site from suspicious, so that it does not prevent our customers from purchasing goods.

**Comment by sbaisarov** [comment]
There is a possibility that your website violates Discord's ToS, despite of your fair sale of accounts in question

**Comment by mmoadvert** [comment]
Our site has already been unblocked everywhere where there were erroneous blockings, including crypto wallets. 

TOS is the rules of the social network, which regulates users, but can not influence or prohibit anything on other sites. 

I don't see you having hundreds of thousands of other stores that sell social media accounts blocked. I guess they have TOS too.

**Comment by sbaisarov** [comment]
I agree with that, your activity has nothing to do with crypto security issues.

The website of this project isn't working. I think the main developer has kinda been missing for now. The only option is to wait.
